includes/functions/llms-functions-user-information-fields.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Functions for LifterLMS user information fields
+ *
+ * @package LifterLMS/Functions
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Retrieve a single user information field by it's ID attribute.
+ *
+ * @since [version]
+ *
+ * @param string $id The field's id
+ * @return array|boolean Returns the field settings array or `false` when the field cannot be found.
+ */
+function llms_get_user_information_field( $id ) {
```

Comment:
I don't know if the `id` is the correct field's field to look for, I mean, at a certain point we actually decided that the "unique" field was the `name` and not the `id`, and you implemented a js code to prevent duplicates.

Reply:
good call